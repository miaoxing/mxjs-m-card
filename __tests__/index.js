import Card from '..';
import {render} from '@testing-library/react';

describe('Card', () => {
  test('basic', () => {
    const {container} = render(<Card/>);
    expect(container).toMatchSnapshot();
  });
});
