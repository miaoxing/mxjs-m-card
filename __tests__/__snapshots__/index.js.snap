// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card basic 1`] = `
<div>
  <taro-view-core
    class="mx-card"
  />
</div>
`;
